requirements/tests.ini

```diff
@@ -1,7 +1,7 @@
 -r base.txt
 
 coverage
-pycodestyle==2.5.0
+pycodestyle==2.8.0
```

Comment:
I have the feeling that this package is also not used. flake8 should already install it as its sub-dependency automatically if I'm not mistaken.

Reply:
I've checked and this is true, I removed the explicit installation of pycodestyle